containers/jupyterlab/Dockerfile

```diff
@@ -17,30 +17,15 @@
 ARG JUPYTERLAB_VERSION=0.35.6
 
 # Image for building ready-to-distribute plugin frontend
-FROM node:10.15.3-stretch AS plugin-frontend-builder
```

Comment:
Are you sure that this will no slow building time?

Reply:
What do you mean exactly?